wrap/pybind11/.github/workflows/format.yml

```diff
@@ -0,0 +1,19 @@
+name: Format
```

Comment:
Delete appveyor files. And what is this format file? Again:merge in develop after landing latest CI changes?

Reply:
This is from the pybind11 subtree :)